From f6364fa0e0c5531f365f0f4a6d4c640100087304 Mon Sep 17 00:00:00 2001
From: Bai Ping <b51503@freescale.com>
Date: Tue, 13 Jan 2015 22:19:53 +0800
Subject: [PATCH 0558/5242] MLK-10089 arm: imx: Add 198MHz OPP for i.MX6SX

commit  aa46c49e4ef8ff7b31dd6f44f5281475a4fb05c6 from
https://source.codeaurora.org/external/imx/linux-imx.git

Update the i.MX6SX operating points to comply with the latest
datasheet. Latest i.MX6SX datasheet of Rev.F, 1/2015 adds the
198MHz setpoint. For the RevB board, the VDD_ARM and ADD_SOC
are connected together, so the voltage for 198MHz needs to be
set to 1.175V. for the general setting, add a 25mV margin to
cover the board IR drop.

Signed-off-by: Bai Ping <b51503@freescale.com>
(cherry picked from commit 62bd7207e6346f404589ed8305971c9815d2cf8b)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sx-sdb-ldo.dts  |    2 ++
 arch/arm/boot/dts/imx6sx-sdb-reva.dts |    2 ++
 2 files changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/imx6sx-sdb-ldo.dts b/arch/arm/boot/dts/imx6sx-sdb-ldo.dts
index 0bdb7f6..3a8c194 100644
--- a/arch/arm/boot/dts/imx6sx-sdb-ldo.dts
+++ b/arch/arm/boot/dts/imx6sx-sdb-ldo.dts
@@ -14,12 +14,14 @@
 		996000  1250000
 		792000  1175000
 		396000  1075000
+		198000	975000
 		>;
 	fsl,soc-operating-points = <
 		/* ARM kHz      SOC uV */
 		996000	1175000
 		792000	1175000
 		396000	1175000
+		198000	1175000
 	>;
 	fsl,arm-soc-shared = <0>;
 };
diff --git a/arch/arm/boot/dts/imx6sx-sdb-reva.dts b/arch/arm/boot/dts/imx6sx-sdb-reva.dts
index a002a4e..5dbf130 100644
--- a/arch/arm/boot/dts/imx6sx-sdb-reva.dts
+++ b/arch/arm/boot/dts/imx6sx-sdb-reva.dts
@@ -18,12 +18,14 @@
 		996000  1250000
 		792000  1175000
 		396000  1075000
+		198000	975000
 		>;
 	fsl,soc-operating-points = <
 		/* ARM kHz      SOC uV */
 		996000	1175000
 		792000	1175000
 		396000	1175000
+		198000	1175000
 	>;
 	fsl,arm-soc-shared = <0>;
 };
-- 
1.7.9.5

